src/templates/aws/advanced.ts

```diff
@@ -112,51 +111,190 @@ export default class Advanced {
 
     const ssmModuleContent = dedent`
     module "ssm" {
-      source = "../modules/ssm"
-    
+      source = "./modules/ssm"
+
       namespace = var.namespace
       secret_key_base       = var.secret_key_base
-    
+
       aws_access_key_id     = var.aws_access_key_id
       aws_secret_access_key = var.aws_secret_access_key
-    
+
       rds_username      = var.rds_username
       rds_password      = var.rds_password
       rds_database_name = var.rds_database_name
       rds_endpoint      = module.db.db_endpoint
     }`
 
-    injectToFile('main.tf', ssmModuleContent, this.options, {insertAfter: '# SSM'})
+    injectToFile('main.tf', ssmModuleContent, this.options, {
+      insertAfter: '# SSM',
+    })
   }
 
   private applySecurityGroup(): void {
-    copyDir('aws/modules/security_group', 'modules/security_group', this.options)
+    copyDir(
+      'aws/modules/security_group',
+      'modules/security_group',
+      this.options,
+    )
 
     const ssmVariablesContent = dedent`
-    variable "app_port" {
-      description = "Application running port"
-      type        = number
-    }
-    
     variable "nimble_office_ip" {
       description = "Nimble Office IP"
     }\n\n`
     appendToFile('variables.tf', ssmVariablesContent, this.options)
 
     const ssmModuleContent = dedent`
     module "security_group" {
-      source = "../modules/security_group"
-    
+      source = "./modules/security_group"
+
       namespace                   = var.app_name
```

Comment:
please also help me to fix this to be consistent with previuos modules in this same file
```suggestion
      namespace                   = var.namespace
```